add-slb-nncp.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: nmstate.io/v1                                                       
+kind: NodeNetworkConfigurationPolicy                                            
+metadata:                                                                       
+  name: slb
+spec:
+  capture:
+    primary-nic: interfaces.description == "primary"
+    secondary-nic: interfaces.description == "secondary"
+  desiredState:
+    interfaces:
+    - name: brcnv-iface
+      type: ovs-interface
+      state: up
+      mac-address: "{{ capture.primary-nic.interfaces.0.mac-address }}"
+      ipv4: "{{ capture.primary-nic.interfaces.0.ipv4 }}"
```

Comment:
We can be explicit here and set it to DHCP. Just to copy exactly what we did with the original script